### PR TITLE
Update caniuse/demos

### PR DESCRIPTION
**Pull request by alice**
@cookiecrook @dot-miniscule please take a look?

### CODE REVIEW

File: demos/index.html [hunk 1]
```diff
@@ -518,101 +518,52 @@ <h3>
...
-        Some users may not be comfortable revealing this, due to concerns of discrimination, or of being offered an undesired alternate
-        interface.
+        New input event types capturing common patterns will also be added, 
+        and where supported will be triggered by user interactions which trigger 
+        the events which would be fired as a fallback.
       </p>
-      <p>
-        So triggering an AOM event listener brings up a new
-        <b>user permission</b> dialog before the web page can capture the event.
+      <p>Once supported everwhere,
```
Comment (dot-miniscule) [inline demos/index.html]: sp: everywhere

File: demos/index.html [hunk 2]
```diff
@@ -518,101 +518,52 @@ <h3>
...
+        Phase 2: Accessible Actions / Input Events
       </h3>
       <p>
-        If an AOM event listener is ever called, the web page *definitively knows* that the user is running AT.
-      </p>
-      <p>
-        Some users may not be comfortable revealing this, due to concerns of discrimination, or of being offered an undesired alternate
-        interface.
+        New input event types capturing common patterns will also be added, 
+        and where supported will be triggered by user interactions which trigger
```
Comment (dot-miniscule) [inline demos/index.html]: nit: this sentence is hard to follow, can it be reworded at all?

File: demos/index.html [hunk 3]
```diff
@@ -518,101 +518,52 @@ <h3>
         Phase 2: Accessible Actions / Input Events
       </h3>
       <p>
-        Listen to input events directly from assistive technology.
+        To preserve the privacy of assistive technology users,
```
Comment (dot-miniscule) [inline demos/index.html]: Is it worth putting a link to the discussion on this topic, seeing as it got such a strong reaction? I'm also not sure I understand the use of the word typically in this case. Like, is it typically because that's the current norm? Because it only does it under certain circumstances?

File: caniuse.md [hunk 4]
```diff
@@ -4,67 +4,138 @@ Track the implementation status of the AOM in various browsers.
...
-element.accessibleNode.label = "Click Me";
-element.accessibleNode.labeledBy = new AccessibleNodeList();
-element.accessibleNode.labeledBy.add(other.accessibleNode);
-...
-```
 
-### Phase 2: Listen for input events from AT.
+### Phase 1: Reflect element references for IDREF attributes
+
+*Not yet implemented*
```
Comment (dot-miniscule) [inline caniuse.md:45]: nothing review related just wanted to leave a :grimacing: face here.

**Review by dot-miniscule — APPROVED** [review]
lgtm w/ a few nits, thanks!

File: caniuse.md (hunk 4 above)
Comment (alice) [reply caniuse.md:45]: 😬

File: demos/index.html (hunk 3 above)
Comment (alice) [reply demos/index.html]: Link added. 

Not sure where "typically" came from to be honest; reworded to "... cause a synthesised DOM event to be triggered where appropriate".

File: demos/index.html (hunk 2 above)
Comment (alice) [reply demos/index.html]: Thanks for catching :) Reworded, and I added another paragraph for an example.

File: demos/index.html (hunk 1 above)
Comment (alice) [reply demos/index.html]: Had to delete this paragraph anyway to fit in the example, but thanks for catching :)
